Modules/System/SharePoint/src/helpers/SharePointUriBuilder.Codeunit.al

```diff
@@ -104,25 +110,37 @@ codeunit 9110 "SharePoint Uri Builder"
     procedure GetUri(): Text
     var
         FullUri: Text;
+        ParameterName, ParameterValue : Text;
     begin
         FullUri := UriLbl + Uri + '/';
         FullUri := FullUri.Replace('{server_name}', ServerName.TrimStart('/').TrimEnd('/')).Replace('{namespace}', Namespace.TrimStart('/').TrimEnd('/'));
+        if QueryParameters.Count() > 0 then begin
```

Comment:
Why aren't you using the URI Builder Codeunit, that supports QueryParameters?
https://github.com/microsoft/ALAppExtensions/blob/6c169363defaf6193dac3cf5f00bba83b06307de/Modules/System/URI/src/UriBuilder.Codeunit.al#L189

Please make sure that there is an dependency to the URI Module. https://github.com/microsoft/ALAppExtensions/blob/main/Modules/System/URI/app.json